include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -180,6 +181,12 @@ ORT_API_STATUS(OrtCreateEnvWithCustomLogger, OrtLoggingFunction logging_function
                _In_ const char* logid,
                _Out_ OrtEnv** out);
 
+
+/**
+ * \load a model from disk and return an opaque handle for it.
+ */
+ORT_API_STATUS(OrtLoadModel, _In_ const ORTCHAR_T* model_path, _Out_ OrtModel** model_handle);
```

Comment:
`OrtCreateModel` ? Since this follows the pattern of the other OrtCreate* functions. Unless you think there'll be a way to create a model in other ways in the future?